modules/vpc/CHANGELOG.md

```diff
@@ -1 +1,3 @@
 # Changelog
+TF version update to 1.9.3
+Remove zone and edit naming convention
```

Comment:
Following the format in https://github.com/frontierhq/azurerm-terraform-modules/blob/main/modules/virtual-network/CHANGELOG.md, this should be:
```suggestion

## 2.0
* TF version update to 1.9.3.
* Naming convention changed.
* `zone` variable removed.
```